config_gen.py

```diff
@@ -118,14 +118,9 @@ def main():
                 c_count = 0
 
             if(c_count == 0 and cxx_count == 0):
-                print("")
+                print()
                 print("ERROR: No commands were logged to the build logs (C: {}, C++: {}).".format(c_build_log.name, cxx_build_log.name))
                 print("Your build system may not be compatible.")
-
-                if(not args["verbose"]):
-                    print("")
-                    print("Try running with the --verbose flag to see build system output - the most common cause of this is a hardcoded compiler path.")
-
```

Comment:
It looks like you've overwritten some newer changes - I think you must have copied and pasted your modified file over the more recent version?

To undo this, make a backup of the file, then run `git reset --hard develop` to undo the changes. Then copy the file back, and commit *only the changes you have made*. (Use `git gui` or `git add -p` for this.)